anitya/lib/utilities.py

```diff
@@ -67,13 +68,27 @@ def check_project_release(project, session, test=False):
     versions_prefix = []
     max_version = ''
 
+    # don't change actual data during test run
+    if not test:
+        project.last_check = datetime.now()
```

Comment:
``arrow.utcnow().datetime`` I think.

Reply:
I had to use `arrow.utcnow().naive`. Otherwise the python 2.7 tests are starting to fail.